qtpy/QtCharts.py

```diff
@@ -4,38 +4,36 @@
 # Licensed under the terms of the MIT License
 # (see LICENSE.txt for details)
 # -----------------------------------------------------------------------------
+
 """Provides QtChart classes and functions."""
 
-# Local imports
-from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, PythonQtError
+from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, PythonQtError, QtModuleNotInstalledError
 
 if PYQT5:
     try:
         from PyQt5.QtChart import *
         from PyQt5 import QtChart as QtCharts
-    except ImportError as error:
-        raise PythonQtError(
-            'The QtChart module was not found. '
-            'It needs to be installed separately for PyQt5.'
-            ) from error
+    except ModuleNotFoundError as error:
+        raise QtModuleNotInstalledError(
+            name='Qt3DCharts', missing_package='PyQtChart'
```

Comment:
Maybe a copy-paste mistake? I think the module is QtCharts/QtChart (without the 3D)
```suggestion
            name='QtCharts', missing_package='PyQtChart'
```